src/components/filterTag/FilterTags.jsx

```diff
@@ -1,13 +1,14 @@
+import { Button } from "@chakra-ui/react";
 import { withSearch } from "@elastic/react-search-ui";
 import React from "react";
 import "./filterTags.scss";
 
 const FilterTags = ({ filters, addFilter, removeFilter, field, options }) => {
   const onRemove = (value) => {
-    removeFilter(field, value, "all");
+    removeFilter(field, value, "any");
```

Comment:
this is where I changed the filter behaviour. You can revert it so the sidebar and filter pills both have the same filter behaviour

Reply:
Oh seen, would resolve now